### PR TITLE
fix(SelectList): overwrite styling with enforced color scheme for menu

### PR DESCRIPTION
**Pull request by JanHamara**
## What

Fixing the problem from Issue #434 

### Media

https://www.loom.com/share/a76665438d7745adb3abeb1cf962e7b6?sid=73419f0a-b715-4d97-afa9-e3fc575b82d3

## How

- Use `useClosestColorScheme` to identify if enforced color scheme is applied
- Wrap the `Menu` component from `react-windowed-select` with respective color scheme component

### CODE REVIEW

File: src/components/SelectList/SelectList.tsx [hunk 1]
```diff
@@ -309,21 +311,54 @@ const MultiValueRemove = props => (
     </ReactSelectComponents.MultiValueRemove>
 );
 
+// eslint-disable-next-line react/prop-types
```
Comment (martimalek) [inline src/components/SelectList/SelectList.tsx:314]: Doesn't `react-select` export props for the `Menu` component? If they exist we could get rid of both the eslint and the ts comment

**Review by martimalek — APPROVED** [review]
Nice job! 🚀 

I've just left one comment to see if we can type some props and get rid of some warnings... 🤔

File: src/components/SelectList/SelectList.tsx (hunk 1 above)
Comment (JanHamara) [reply src/components/SelectList/SelectList.tsx]: good point 👍

File: src/components/SelectList/SelectList.tsx [hunk 2]
```diff
@@ -309,21 +312,48 @@ const MultiValueRemove = props => (
     </ReactSelectComponents.MultiValueRemove>
 );
 
+const LightSchemeMenu = (children, props: WithSelectProps<MenuProps>) => (
+    <LightScheme>
+        <ReactSelectComponents.Menu {...props}>{children}</ReactSelectComponents.Menu>
+    </LightScheme>
+);
+
```
Comment (martimalek) [inline src/components/SelectList/SelectList.tsx:320]: Is there a need to have children as a first argument? I believe it should be a regular prop 🤔 

```suggestion
const LightSchemeMenu = (props: WithSelectProps<MenuProps>) => (
    <LightScheme>
        <ReactSelectComponents.Menu {...props}>{props.children}</ReactSelectComponents.Menu>
    </LightScheme>
);

```

**Pull request merged**